client/platform/web-girder/views/Export.vue

```diff
@@ -14,32 +23,68 @@ export default {
     },
   },
 
-  data() {
-    return {
-      menuOpen: false,
-      excludeFiltered: false,
-      activator: 0,
-    };
-  },
+  setup(props) {
+    const menuOpen = ref(false);
+    const excludeFiltered = ref(false);
+    const exportUrls = ref(null as null | ExportUrlsResponse);
+    const savePrompt = ref(false);
+    const currentSaveUrl = ref('');
+    let save = () => Promise.resolve();
+    let pendingSaveCount = ref(0);
+
+    try {
+      save = useHandler().save;
+      pendingSaveCount = usePendingSaveCount();
+    } catch (err) {
+      /**
+       * Error indicates that this export dialog was opened in a place
+       * outside the viewer where pending changes aren't possible.
+       */
+    }
```

Comment:
Still going to get a Vue Warn on this obviously if in Home.vue.  It's not the best indicator but the `prop.small` is only true when in  `Home.Vue` so it may be used as a check against trying to access `use(Handler|PendingSaveCout)`.  It doesn't feel clean but could be used as additional check as well as catching the error.

Reply:
I got lazy.   I don't like having weird side-effects based on seemingly unrelated props, so I added a new more explicit one.

Reply:
My long winded response was mostly because I felt weird using `small` to indicate that.  A more explicit prop is a better idea.